guidance_and_support/tests/test_zendesk.py

```diff
@@ -52,3 +52,40 @@ def test_generate_ticket(user):
     """Test a ticket from a valid user and a spam bot."""
     ticket = generate_ticket(user['request'], user['form'], score=user['score'])
     assert ticket == user['expected_output']
+
+
+@pytest.mark.django_db
```

Comment:
Thanks for tests! As they change `LEGITIMATE_USER` global object however, if these new tests are ever run before the `test_generate_ticket` test then the global object will be changed and the original test will never be done.
Maybe in your tests you could copy the `LEGITIMATE_USER` object with  https://docs.python.org/3/library/copy.html#copy.deepcopy and then change and use the copy?

Reply:
Thanks @odscjames!